### PR TITLE
Upgrade Quarkus to 2.1.1.Final

### PR DESCRIPTION
**Pull request by jamesnetherton**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

**Comment by jamesnetherton** [comment]
For anyone wondering about the formatting changes in `pom.xml`, it comes from running `cq:sync-versions`. Figured it was about time to commit them....

**Review by zbendhiba — APPROVED** [review]

**Review by aldettinger — APPROVED** [review]

**Review by ppalaga — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -585,7 +585,7 @@
                 <executions>
                     <execution>
                         <id>default-test</id>
-                        <phase /><!-- unbound here to speedup -Dquickly build; bound to a phase in camel-quarkus-build-parent-it -->
```
[the remark this answers is not in the document]
Comment (jamesnetherton) [reply pom.xml:588]: No, it was modified in 09935b880028899fc7992a11e65e728a622ecfa5.

[the remark this answers is not in the document]
Comment (ppalaga) [reply pom.xml:588]: Ups, sorry for accusing you! Yeah, the spaces are added by `mvn release:update-versions`. Let me have a look what we can do.

[the remark this answers is not in the document]
Comment (ppalaga) [reply pom.xml:588]: jdom2 used by the release plugin internally is very opinionated about this: https://github.com/hunterhacker/jdom/blob/master/core/src/java/org/jdom2/output/support/AbstractXMLOutputProcessor.java#L824 So I better adapt the cq plugin.

**Pull request merged**